docker/Dockerfile.cpu

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:16.04
+
+# MAINTAINER sshuair<sshuair@gmail.com>
+
+# install dependencies
+RUN apt-get update --fix-missing && apt-get install -y --no-install-recommends\ 
+        build-essential \
+        software-properties-common \
+        curl \
+        cmake \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        libspatialindex-dev \
+        pkg-config \
+        rsync \
+        zip \
+        unzip \
+        git \
+        wget \
+        vim \
+        ca-certificates \
+        python3 \
+        python3-dev \
+        python3-pip \
+        ipython3 \
+        libboost-python-dev \
+        libexpat1-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libsm6 \
+        libxext6 \
+        libxrender1 \
```

Comment:
Why are all these dependencies needed?

Reply:
if not install these dependencies, it will report this error:
-  `ImportError: libSM.so.6: cannot open shared object file: No such file or directory` and 
- `ImportError: libXrender.so.1: cannot open shared object file: No such file or directory`

I will test the library that is not necessary later.